### ISSUE TITLE
Godot CollisionShape2D and CollisionPolygon2D never seem to line up properly with my tiles

### ISSUE THREAD

**Issue by SleepE** [godotengine/godot#24290]
**Godot version:**
Godot 3.0

**OS/device including version:**
Windows 10 64 bit

**Issue description:**
Whenever I go to make a tile-set (tres) with collision the collisions never seem to line up properly with my tiles, and never seem to be the same as when I set them up when making the tile-set. I know I must be doing something wrong but I am not sure what. Here I have a small recreation of the issue so that you can see what I mean.

I have a scene here that I've turned into a tile-set, with just 2 tiles and CollisionShape2D rectangles (I have tried using CollisionPolygon2D).

![collision1](https://user-images.githubusercontent.com/40583710/49837290-dcaa1300-fd73-11e8-942a-7d82d0adf22f.PNG)

![collision2](https://user-images.githubusercontent.com/40583710/49837306-e5024e00-fd73-11e8-8f3a-c18f09ecdd1d.PNG)

Now I have a root scene here, where I have a player with some collisions, he is supposed to fall on the tiles that I've drawn and stop on top of them.

![collision3](https://user-images.githubusercontent.com/40583710/49837332-f9dee180-fd73-11e8-98f8-3e2802e6ceb4.PNG)

However, when I run the game, this happens to the collisions.

![collision4](https://user-images.githubusercontent.com/40583710/49837352-09f6c100-fd74-11e8-9259-33ee1f1471bd.PNG)

I have tried making sure that the scale is set to (1, 1) as I've seen suggested in other places. I have also tried doing this in many ways, and with a few different types of games. It never seems to work. I once was able to get it functioning properly, but I was forced to scale the Collisions down to 1/2 of their size and misalign them a little bit. However, this can be annoying and a bit time consuming and I'm wondering if there is something wrong that I am doing?

Thank you for reading and sorry for this long and probably stupid post.

**Comment by girng** [comment]
have you tried saving the tileset with merge disabled? sometimes it causes wonky stuff, although not sure if this could be one

**Comment by bojidar-bg** [comment]
I'm pretty sure this is a duplicate of #19956. Make sure to check the comments there as well.

TL;DR: ensure that the StaticBody2D node has scale = (1,1) and position = (0,0) in the inspector.

Closing as duplicate. Additionally, it should have been fixed in 3.1.
To be reopened only if it can be confirmed in 3.1/master as well (make backups though, as project.godot format changed).
